examples/loco-rooms-chat/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "loco_chat_rooms"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+
+loco-rs = { version = "0.2.3", default-features = false, features = [
+    "cli",
+    "channels",
+] }
+serde = { version = "1.0.195", features = ["derive"] }
+serde_json = "1.0.111"
+eyre = "0.6.11"
+tokio = { version = "1.33.0", default-features = false }
+async-trait = "0.1.74"
+chrono = { version = "0.4", features = ["serde"] }
+
+axum = "0.7.1"
+tracing = "0.1.40"
+tracing-subscriber = { version = "0.3.17", features = ["env-filter", "json"] }
+
```

Comment:
For the dependencies that are already defined in the main Cargo.toml could you link to them with `workspace = true` ? It would allow to have automatic update from dependabot and to check that this example works with the new versions.
Specified deps are tokio, axum, tracing, tracing-subscriber, serde and serde_json